extensions/oplogPopulator/modules/ConnectorsManager.js

```diff
@@ -263,11 +265,7 @@ class ConnectorsManager extends EventEmitter {
                 });
                 return true;
             } else if (connector.isRunning && this._allocationStrategy.canUpdate()) {
-                const isPipelineUpdated = connector.updatePipeline(true);
-                if (isPipelineUpdated) {
-                    this.emit('connector-updated', connector);
-                }
-                return isPipelineUpdated;
+                return connector.updatePipeline(true);
```

Comment:
still need to `await`, otherwise the return value is not a bool ?

Reply:
No need to await a function we return in an async function, if we await it (the parent function), then this line will be awaited and return a bool